### PR TITLE
Event submission Cypress test

### PR DESCRIPTION
**Pull request by MatthewGidcomb**
[Infinite #11](https://github.com/infinite-industries/infinite/issues/11)

The one piece of this I'm not happy with is the way the key pair for overriding auth is handled. I couldn't get `fs` to work in the Cypress code, so I had to use the Cypress fixtures system to load the key to sign the JWTs. That means the key needs to be in `cypress/fixtures/` instead of `keys/`.
Also we talked about this last week but still not sure we should check in the key pair, even though we know they can't compromise anything.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,9 +77,22 @@ SLACK_WEBHOOK_EVENT_SUBMISSION=################################################
 
 ...
 ```
+See also .env.sample
 
 Follow the sample template in src/clientConfig.sample.js to setup client settings for authO
 
+## Running Tests
```
Comment (chriswininger) [inline README.md]: You also need to have the api service running

File: cypress/integration/event_submit_process.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
+describe('Testing event submission flow', function() {
```
Comment (chriswininger) [inline cypress/integration/event_submit_process.js]: The event submit spec fails for me with .card h3, Test Event
expected body .ard
h3:not(script):contains(Test Event). .card h3[type='submit'][value='Test Event'] to exist in the DOM

TypeError: Path must be a string. Recevied null

It appears to fail during the admin verification. In fact the app does not appear to be logged in; so, maybe it's not using the key file or setting up the token or something?

File: package.json [hunk 3]
```diff
@@ -6,9 +6,10 @@
   "scripts": {
     "start": "node server.js",
     "start-dev": "concurrently \"npm run watchify\" \"nodemon ./server.js\"",
+    "start-test": "concurrently \"npm run watchify\" \"SECRET_STRING_FILE=\"./cypress/fixtures/keys/1nfinite_testing.pem\" nodemon ./server.js\"",
```
[the remark this answers is not in the document]
Comment (MatthewGidcomb) [reply package.json]: Looks like `npm run ...` doesn't yield for the next command after the `&&` to run, but it works with concurrently `concurrently "npm run start-test" "npm run test-open"`.

File: cypress/integration/event_submit_process.js (hunk 2 above)
Comment (MatthewGidcomb) [reply cypress/integration/event_submit_process.js]: I forgot to add `cp cypress.env.json.sample cypress.env.json` to the test setup instructions; added now.

**Pull request merged**